recon/src/recon/tests.rs

```diff
@@ -954,20 +1013,28 @@ fn message_cbor_serialize_test() {
         .into();
     let received_cbor = hex::encode(serde_ipld_dagcbor::ser::to_vec(&received).unwrap());
     println!("serde_json {}", serde_json::to_string(&received).unwrap()); // Message as json
-    expect![["a2616b826161616361688158203e23e8160039594a33894f6564e1b1348bbd7a0088d42c4acb73eeaed59c009d"]].assert_eq(&received_cbor);
+    expect![["a2616b824161416361688158203e23e8160039594a33894f6564e1b1348bbd7a0088d42c4acb73eeaed59c009d"]].assert_eq(&received_cbor);
```

Comment:
Note: This is the only wire protocol change. The CBOR is Byte stings not Utf-8 strings.